Makefile

```diff
@@ -125,14 +125,20 @@ endif
 ENABLE_JIT ?= 0
 $(call set-feature, JIT)
 ifeq ($(call has, JIT), 1)
-OBJS_EXT += jit.o
+OBJS_EXT += jit.o t2jit.o
+CFLAGS += -g `llvm-config${VER} --cflags`
```

Comment:
T2C is only built when the supported versions of LLVM packages are available. That is, T2C is ignored if the LLVM check cannot be satisfied.